src/azure-cli/azure/cli/command_modules/appconfig/_params.py

```diff
@@ -81,6 +81,7 @@ def load_arguments(self, _):
 
     with self.argument_context('appconfig create') as c:
         c.argument('location', options_list=['--location', '-l'], arg_type=get_location_type(self.cli_ctx), validator=get_default_location_from_resource_group)
+        c.argument('tags', arg_type=tags_type, help="Space-separated tags: key[=value] [key[=value] ...].")
```

Comment:
The help has been defined in `tags_type`, why do you want to define it again?

Reply:
The default help message in ```tags_type``` doesn't apply to create command. It says ```Use "" to clear existing tags```, which is suitable for update only.

Reply:
OK~